src/ik.rs

```diff
@@ -52,10 +52,11 @@ where
     diff
 }
 
-/// true means the constraint is used.
-///  The coordinates is the world, not the end of the arm.
+/// A bundle of flags determining which coordinates are constrained for a target
```

Comment:
The primary reason for this is the "true means..." was showing up as the description on the library page.

![2019-07-11_10-22-09](https://user-images.githubusercontent.com/4165105/61058738-cde91480-a3c5-11e9-8850-d6cb878de256.png)